### ISSUE TITLE
Handlebars Dependency Broken

### ISSUE THREAD

**Issue by adamisntdead** [japaric/criterion.rs#137]
I am getting a build error 

```
error[E0518]: attribute should be applied to function
 --> /Users/travis/.cargo/registry/src/github.com-1ecc6299db9ec823/handlebars-0.31.0/src/grammar.rs:3:10
  |
3 | #[derive(Parser)]
  |          ^^^^^^ not a function
error: aborting due to previous error
For more information about this error, try `rustc --explain E0518`.
error: Could not compile `handlebars`.
```

which I believe is being caused by handlebars

**Comment by adamisntdead** [comment]
I should mention that that is on nightly rust, happy to fix and do a pull request if needed

**Comment by bheisler** [comment]
Thanks for the report, and for trying Criterion.rs.

I'll try to fix this soon. A pull request would be welcome if you get around to it first, though.

**Comment by adamisntdead** [comment]
I keep getting the error on travis, but I cannot seem to reproduce it on my local machine... https://travis-ci.org/QCGPU/qcgpu-rust/jobs/361201489

**Comment by adamisntdead** [comment]
Now also failing on nightly, any ideas how to fix?

Thanks!

**Comment by bheisler** [comment]
According to https://github.com/sunng87/handlebars-rust/issues/209, the problem is solved by the latest version of pest (1.0.7). `cargo update` should fix it - it seems to have worked for me, at least.

**Comment by adamisntdead** [comment]
That worked for me thanks!
